_posts/2022-10-18-cryosat-blog.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: '"Archive On Stop" feature in the Cryostat 2.2'
+date: 2022-10-18
+synopsis: Cryostat 2.2 New features
+author: Leticia Konno
+
+---
+
+Hello, readers!
+
+There is a new option by default for Custom Flight Recording creation to have the recording automatically archived on stop. 
+
+When manually capturing a new recording in the **Recordings** tab, it is possible to select if it is going to be *Continuous* or with a fixed duration of time. Create a recording with a defined duration and keep the **Archive On Stop** option checked as below:
+
+![Alt text](/images/archive-on-stop-1-post.png?raw=true "Archive On Stop option")
```

Comment:
Do you think we should have a specific alt-texts for each image? It should help in accessibility support :D

Reply:
I suppose you mean for other images on the site, since it seems the images in this post do have alt-texts?

Reply:
oh I mean a more descriptive alt-text instead of just plain Alt text.

Reply:
Oh from second look, didnt realize this way of adding alt-text. But the new alt-text seems better too :D